packages/fxa-react/lib/test-utils/index.ts

```diff
@@ -67,6 +67,32 @@ async function getFtlFromPackage(packageName: PackageName, locale: string) {
         );
       }
       break;
+    case 'shared':
+      if (locale === 'en') {
+        ftlPath = path.join(
+          __dirname,
+          '..',
+          '..',
+          '..',
+          'fxa-react',
+          'test',
+          'shared.ftl'
+        );
+      } else {
+        // TODO: Not currently used, but probably want to add one translation test
+        ftlPath = path.join(
+          __dirname,
+          '..',
+          '..',
+          '..',
+          'fxa-react',
+          'public',
+          'locales',
+          locale,
+          'shared.ftl'
+        );
+      }
+      break;
```

Comment:
Note that if your PR is merged before Dan's (#14557), he'll need to update [this line](https://github.com/mozilla/fxa/pull/14557/files#diff-2a5e551ad7a3b664bd3f6fd4f8eba5bffbb8ad378ddd54474af18ded2799c21cR24) to include 'shared', and if his is merged before yours, you'll need to update it.

Reply:
@dschom Do you think this PR should land first or yours? Currently, we're blocked from landing new Fluent strings because of the duplication issue.

Reply:
I am about ready to r+ this, and am still waiting on review for my other PR. If this PR gets approved before mine, feel free to land it first.

Reply:
@dschom Bryan has approved the l10n PR, holding on merging until this PR is merged in. Would you be willing to r+/approve this PR?